rllib/connectors/util.py

```diff
@@ -11,21 +10,29 @@
 from ray.rllib.connectors.agent.pipeline import AgentConnectorPipeline
 from ray.rllib.connectors.agent.state_buffer import StateBufferConnector
 from ray.rllib.connectors.agent.view_requirement import ViewRequirementAgentConnector
-from ray.rllib.connectors.connector import Connector, ConnectorContext, get_connector
+from ray.rllib.connectors.connector import Connector, ConnectorContext
 from ray.rllib.connectors.agent.mean_std_filter import (
     MeanStdObservationFilterAgentConnector,
     ConcurrentMeanStdObservationFilterAgentConnector,
 )
 from ray.rllib.utils.typing import TrainerConfigDict
 from ray.util.annotations import PublicAPI, DeveloperAPI
 from ray.rllib.connectors.agent.synced_filter import SyncedFilterAgentConnector
-
-if TYPE_CHECKING:
-    from ray.rllib.policy.policy import Policy
+from ray.rllib.utils.deprecation import Deprecated
 
 logger = logging.getLogger(__name__)
 
 
+@Deprecated(
+    old="ray.rllib.connectors.util.create_connectors_for_policy",
```

Comment:
This avoids some circular imports.

Reply:
nice. let's just remove this and call it a day?
the connector API was Alpha all-along, plus this is definitely one of the more internal facing util function that nobody should use.